### PR TITLE
IOTX integration

### PR DESCRIPTION
**Pull request by dustinxie**
Closes https://github.com/TrustWallet/blockatlas/issues/101

### CODE REVIEW

**Comment by terorie** [comment]
Thanks for your contribution! Is it possible to add support without introducing a dependency to `github.com/iotexproject`?
We try to keep coin-specific dependencies at a minimum for performance and security reasons (e.g. import times and package side-effects).

**Comment by dustinxie** [comment]
> Thanks for your contribution! Is it possible to add support without introducing a dependency to `github.com/iotexproject`?
> We try to keep coin-specific dependencies at a minimum for performance and security reasons (e.g. import times and package side-effects).

Removed in latest push, pls take a look, thx!

File: platform/iotex/api.go [hunk 1]
```diff
@@ -7,8 +7,6 @@ import(
 
 	"github.com/gin-gonic/gin"
 	"github.com/spf13/viper"
-	"github.com/iotexproject/go-pkgs/crypto"
-	"github.com/iotexproject/iotex-address/address"
```
Comment (dustinxie) [inline platform/iotex/api.go]: removed

File: go.mod [hunk 2]
```diff
@@ -3,21 +3,31 @@ module github.com/trustwallet/blockatlas
 go 1.12
 
 require (
-	github.com/davecgh/go-spew v1.1.1
+	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/fsnotify/fsnotify v1.4.7
```
Comment (dustinxie) [inline go.mod]: result of running 'go mod tidy'

**Comment by dustinxie** [comment]
@terorie 
can you please take a look of new update/commit

File: platform/iotex/model.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package iotex
+
+const TransferFee = "10000000000000000"
```
Comment (vikmeup) [inline platform/iotex/model.go]: is it possible to get transfer fee from the transaction object and not to hardcode?

Comment (dustinxie) [reply platform/iotex/model.go]: our transfer fee is fixed
for contract execution it will varying depending on specific inputs

Comment (vikmeup) [reply platform/iotex/model.go]: can we do `gasLimit * gasPrice` in this case? we should avoid hardcoding any values

Comment (dustinxie) [reply platform/iotex/model.go]: fixed in latest push

**Pull request merged**